lib/mix/lib/mix/tasks/deps.loadpaths.ex

```diff
@@ -35,14 +35,11 @@ defmodule Mix.Tasks.Deps.Loadpaths do
     end
 
     unless "--no-load-deps" in args do
-      load_paths =
-        for dep <- all,
-            path <- Mix.Dep.load_paths(dep) do
-          _ = Code.prepend_path(path)
-          path
-        end
-
-      prune_deps(config, load_paths, "--no-deps-check" in args)
```

Comment:
Why are deps no longer pruned? Performance?

Reply:
The commit message has some context but this is not necessary anymore. before we loaded everything in `_build/ENV/lib/*/ebin` but now we only load your deps, so we never load something we should not.